msg/en_US/utils.msg

```diff
@@ -489,32 +489,33 @@ $set 12 MSGCAT_UTIL_SET_LOADDB
-  -v, --verbose                  enable verbose status messages; default: disabled\n\
-  -c, --periodic-commit=COUNT    insertion COUNT for periodic commit; default: no commit during loading\n\
-      --no-oid                   don't use OID\n\
-      --no-statistics            do not update the statistics while loading\n\
-  -s, --schema-file=FILE[:LINE]  create schema from LINE with FILE\n\
-  -i, --index-file=FILE[:LINE]   create index from LINE with FILE\n\
-      --trigger-file=FILE[:LINE] create trigger from LINE with FILE\n\
-  -d, --data-file=FILE           load data with FILE\n\
-  -t, --table=TABLE              table name that is substituted for missing class header in data file\n\
-      --error-control-file=FILE  FILE to control error(s) during loading\n\
-      --ignore-class-file=FILE   input file of class names that skip load\n\
-      --no-user-specified-name   Find classes, serials, and triggers by their object names without their owner names.\n\
-                                 This is to load files generated by cubrid before version 11.2.\n
-
+  -S, --SA-mode                      standalone mode execution\n\
+  -C, --CS-mode                      client-server mode execution\n\
+  -u, --user=ID                      user ID for database access; default: "public"\n\
+  -p, --password=PASS                user password; default: none\n\
+      --data-file-check-only         check syntax for data file without load; default: disabled\n\
+  -l, --load-only                    load data file without prior syntax checking; SA_MODE only\n\
+      --estimated-size=NUMBER        estimated NUMBER of instances; default: 5000\n\
+  -v, --verbose                      enable verbose status messages; default: disabled\n\
+  -c, --periodic-commit=COUNT        insertion COUNT for periodic commit; default: no commit during loading\n\
+      --no-oid                       don't use OID\n\
+      --no-statistics                do not update the statistics while loading\n\
+  -s, --schema-file=FILE[:LINE]      create schema from LINE with FILE\n\
+  -i, --index-file=FILE[:LINE]       create index from LINE with FILE\n\
+      --trigger-file=FILE[:LINE]     create trigger from LINE with FILE\n\
+  -d, --data-file=FILE               load data with FILE\n\
+  -t, --table=TABLE                  table name that is substituted for missing class header in data file\n\
+      --error-control-file=FILE      FILE to control error(s) during loading\n\
+      --ignore-class-file=FILE       input file of class names that skip load\n\
+      --no-user-specified-name       Find classes, serials, and triggers by their object names without their owner names.\n\
+                                     This is to load files generated by cubrid before version    11.2.\n\
+      --multiload-schema-file=FILE   File for loading multiple schema files, The content of this file must be a schema file created with the --split-schema-file option of unloaddb.\n
```

Comment:
The content of this file is a list of schema file names to be used in loaddb. loaddb is executed sequentially from the top.

Reply:
thanks for your comment.
I'll change it.